pkg/apis/acme/v1/types_issuer.go

```diff
@@ -42,6 +42,14 @@ type ACMEIssuer struct {
 	// Only ACME v2 endpoints (i.e. RFC 8555) are supported.
 	Server string `json:"server"`
 
+	// PreferredChain is the chain to use if the ACME server outputs multiple.
+	// PreferredChain is no guarantee that this one gets delivered by the ACME
+	// endpoint.
+	// For example, for Let's Encrypt's DST crosssign you would use:
+	// "DST Root CA X3" or "ISRG Root X1" for the newer Let's Encrypt root CA.
```

Comment:
Explain that this must match the `CommonName` of the first (or is it last?) certificate in the chain, where the first certificate is the chain root CA and the last certificate is the leaf certificate.

Reply:
It returns if any cert's issuer in the chain matches it, ACME does not have to (and LE does not) give the root CA cert back only leaf and intermediate so we look at the issuer field, analog to certbot

Reply:
Added a note

Reply:
Ok. In my mind, it should only match the highest level CA certificate in the bundle and only match a CA certificate which can be chained to the leaf certificate.

Or to put it another way, you should get an error if you happen to enter the CN of the leaf certificate here.
And you should get an error if the bundle contains any certificates which are not involved in a chain of signatures leading to the leaf cert.